string picker_node
string picker_id
string local_storage_node
int32 n_trays
---
string robot_id
bool success
---
string robot_id
float64 eta_picker_node
float64 eta_local_storage_node
